setup/plugins/init/StandaloneUsers.civi-setup.php

```diff
@@ -49,13 +49,50 @@
           'name' => 'everyone',
           'label' => ts('Everyone, including anonymous users'),
           // Provide default open permissions
-          'permissions' => ['CiviMail subscribe/unsubscribe pages', 'make online contributions', 'view event info', 'register for events'],
+          'permissions' => ['CiviMail subscribe/unsubscribe pages', 'make online contributions', 'view event info', 'register for events', 'authenticate with password'],
         ],
         [
           'name' => 'admin',
           'label' => ts('Administrator'),
+          // @todo Would it be better to grant super user powers via
+          // 'all CiviCRM permissions and ACLs' or 'administer CiviCRM'?
           'permissions' => array_keys(\CRM_Core_SelectValues::permissions()),
         ],
+        // The following role is not required and might be better moved into the demo setup.
+        [
+          'name' => 'staff_user',
+          'label' => ts('Staff User'),
```

Comment:
Yeah, I like the idea of a `staff_user` role. Actually, I had some un-posted notes to myself saying: "We need to talk about roles more..." I think we should get this in, so I'll split that off as https://lab.civicrm.org/dev/core/-/issues/4466